js/pastebox.js

```diff
@@ -71,20 +73,20 @@ function PasteBox() {
 
             this._makeBoxBitmap(PASTEBOX, "box", __processBackground, null);
```

Comment:
`PASTEBOX` is undefined here.

Reply:
@meganindya I checked for that in my chrome debugger tools. It is getting the correct value. Here's a ss of the same

![image](https://user-images.githubusercontent.com/62568269/101245418-d36d9a00-3732-11eb-8b2a-a1f820738dd1.png)

Reply:
Sorry my bad. Saw the wrong diff.